index.bs

```diff
@@ -9350,13 +9349,13 @@ Methods</h5>
 				definition with a duplicated key is not allowed.
 
 			3. If the result of <code><a href="http://www.ecma-international.org/ecma-262/6.0/#sec-isconstructor">
-				IsConstructor</a>(argument=<em>processorConstructor</em>)</code>
```

Comment:
None of these should be `<em>`. They're not emphasized, they're italic for stylistic reasons - use `<i>`

Reply:
Done.